cmd/kyma/run/function/function.go

```diff
@@ -130,11 +128,26 @@ func workspaceConfig(path string) (workspace.Cfg, error) {
 	if _, ok := supportedRuntimes[cfg.Runtime]; !ok {
 		return workspace.Cfg{}, fmt.Errorf("unsupported runtime: %s", cfg.Runtime)
 	}
-
+	sourceFile, depsfile, _ := workspace.InlineFileNames(cfg.Runtime)
```

Comment:
Why is the last parameter being ignored? I am afraid I do not know what it represents from the implementation since it has no godocs.

Reply:
It returns a boolean value that represents if the runtime is supported or not which is not needed as its already validated in the 3 lines above.

Reply:
I suspected that.
In that case, I would rethink how this supported runtimes topic is handled. This explicit check outside the function library looks like an implementation detail leak to me, checking the runtime should happen internally and be baked into all public API calls that need it (e.g. return a specific `RuntimeNotSupportedError` value instead of a boolean).

Reply:
I added the line to confirm runtime support. To change the function in hydroform we will need to look at all of its usages which will be best to be handled in a separate issue/pull request. I will definetly raise the topic to our PO during the next sync.

Reply:
I agree, please create a follow up issue to refactor this on hydroform.